test/integration/k8s/otel/k8s_otel_metrics_test.go

```diff
@@ -11,25 +11,13 @@ import (
 // Run it alphabetically first (AA-prefix), with a longer timeout, to wait until all the components are up and
 // traces/metrics are flowing normally
 func TestOTEL_MetricsDecoration_AA_HTTP_ExternalToPod(t *testing.T) {
-	k8s.DoTestHTTPMetricsDecorationExternalToPod(t)
+	k8s.DoWaitForComponentsAvailable(t)
```

Comment:
K8s tests also become simplified, since we don't have to tests the cases between all the possible types of source and destinations. It would be enough just testing a single use case, but I anyway preferred to test client (simple Pod) and servers (complete Deployment) for HTTP and GRPC.